src/calculateRentalCost.js

```diff
@@ -4,7 +4,44 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const discountUpToSix = 20;
+  const discountSevenMore = 50;
+
+  if (days <= 0) {
+    return 'You need to rent for at least one day';
+  }
+
+  if (days <= 2) {
+    return days * pricePerDay;
+  } else if (days >= 3 && days < 7) {
+    return days * pricePerDay - discountUpToSix;
+  }
+
+  return days * pricePerDay - discountSevenMore;
```

Comment:
Prefer using constants for the number of days that determine the discount thresholds. This makes the code more readable and maintainable. Refer to the checklist item 2 for an example.